Gemfile.lock

```diff
@@ -0,0 +1,74 @@
+GEM
```

Comment:
We generally don't submit the lockfile in git. This suggests it's missing from `.gitignore`. Please drop it from this PR.

Reply:
@ekohl , please could we go ahead and merge this PR so that the corresponding BZ can be closed? Or is there anything else that would need to be fixed? Thank you!